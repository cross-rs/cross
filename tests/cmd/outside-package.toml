bin.name = "cross"
args = ["build"]
status = "failed"
fs.cwd = "/"
