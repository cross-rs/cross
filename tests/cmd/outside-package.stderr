[..][cross] warning[..]:[..] unable to get metadata for package
[..][cross] note[..]:[..] Falling back to `cargo` on the host.
[..]error[..]:[..] could not find `Cargo.toml` in `[CWD]` or any parent directory
